include/seqan3/argument_parser/validators.hpp

```diff
@@ -432,10 +481,10 @@ class input_file_validator : public file_validator_base
                 throw parser_invalid_argument(detail::to_string("The file ", file, " does not exist!"));
 
             // Check if file is regular and can be opened for reading.
-            validate_readability(file);
+            this->validate_readability(file);
 
             // Check extension.
-            validate_filename(file);
+            this->validate_filename(file);
```

Comment:
because, the base class is now a template class. 😒

Reply:
Yes, I can do that but than have to copy the constructors for the output file. This kind of diminish the use of a base class to DRY some code. Just because I need to use this to call the correct functions?